ckanext/georchestra/controller.py

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+import six
+
+import ckan.lib.base as base
+import ckan.lib.helpers as h
+import ckan.plugins.toolkit as t
+
+render = base.render
+
+
+class GeorchestraController(base.BaseController):
+
+    def georchestra_logout(self):
+        '''Override logout with geOrchestra's logout'''
+        return h.redirect_to(u'/logout')
+
+    def georchestra_login(self):
+        '''Override login with geOrchestra's login pattern'''
+        referrer = six.text_type(t.request.referrer, encoding='utf-8')
+        url = referrer+u'&login' if '?' in referrer else referrer+u'?login'
```

Comment:
May be safer with `urlparse`, but good enough.

Reply:
+1 for urlparse or string format.

Reply:
OK. Do you know how to set a key-only parameter using urlparse ? I know how to add 'login=true', but just 'login', i don't. Anyway, 'login=true' seems to work.

Reply:
Now using urlparse